### DISCUSSION TITLE
Document

### DISCUSSION THREAD

**Discussion by csb2024** [Concrete-Chrono-Development/chrono-preprocessor#11] [Q&A] [answered]
Is there a manual for the module?

**Comment by mtroemner** [comment, marked as answer]
Given how new this is, there is no manual yet aside from the small amount of information in the readme and the Wiki. We hope to develop a comprehensive manual in the near future. For now the best course of action when questions arise are to post them them and get feedback from one of the users or developers. 

On a side note, I will convert this to a Discussion instead of an Issue. Feel free to follow-up with any additional questions.
